ui/app/models/variable.js

```diff
@@ -97,10 +97,13 @@ export default class VariableModel extends Model {
   get pathLinkedEntities() {
     const entityTypes = ['job', 'group', 'task'];
     const emptyEntities = { job: '', group: '', task: '' };
-    if (this.path?.startsWith('jobs/') && this.path?.split('/').length <= 4) {
+    if (
+      this.path?.startsWith('nomad/jobs/') &&
+      this.path?.split('/').length <= 5
+    ) {
       return this.path
         .split('/')
-        .slice(1, 4)
+        .slice(2, 5)
```

Comment:
`thought` `non-blocking`: There isn't anything to help protect this logic.

Reply:
I think that's okay; splits are non-destructive to the initial string in the event they're not found: 
![image](https://user-images.githubusercontent.com/713991/180026283-0eee307a-7731-48f4-a317-a490fb02a433.png)